src/components/Form.js

```diff
@@ -65,12 +69,17 @@ const Form = () => {
           placeholder="category"
           onChange={(e) => setCategory(e.target.value)}
           style={{
-            width: '25%',
+            width: '27%',
             marginRight: '4%',
-            padding: '1.5%',
+            height: '2.23rem',
+            margin: '1.188rem 2.125rem 0',
+            padding: '0.375rem 0.938rem 0.563rem 1rem',
+            borderRadius: '4px',
+            border: 'solid 1px #e8e8e8',
+            backgroundColor: '#fff',
           }}
         />
-        <select
+        {/* <select
           name="category"
           style={{
             width: '20%',
```

Comment:
- [ ] Why don't you remove all the unnecessary commented lines of code in your codebase so as to achieve a clean and readable codebase? 😄